.github/workflows/pr-check-signed-commits.yml

```diff
@@ -0,0 +1,16 @@
+name: Check signed commits in PR 
+on: pull_request_target
+
+jobs:
+  build:
+    name: Check signed commits in PR 
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
```

Comment:
We actually don't need this step. All the data we need is available from the GitHub API. I'll make sure to also update it in other repos.
```suggestion
```